### DISCUSSION TITLE
Add option to remove sticky post in post widget query

### DISCUSSION THREAD

**Discussion by vgstef** [elementor/elementor#28206] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

In the post widget, we want to show posts for a specific term or do a manual selection for posts. But if there are sticky posts (selected my us or a client), those sticky posts will always show up, even if they're not related to the term or not in the list of manually selected posts. That's very problematic, because it looks like a bug on the websites. 

### Describe the solution you'd like.

We just need a small checkbox saying "Remove sticky posts" in the post widget. 

### Describe alternatives you've considered.

We considered removing all sticky posts on the websites. But on the home page, we need to show a few featured posts (sticky posts). And those posts are chosen by an editor that doesn't have access to the widget query.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by LoganStrike** [comment]
For additional info, reference the decision not to include this feature when it was reported as a bug. This is an essential feature, because the current functionality prohibits users from using the post module for a manual query of a custom post type to display only posts from that custom post type. It should absolutely be considered a bug.

https://github.com/elementor/elementor/issues/18689#issuecomment-2250517738

Users should not have to add custom PHP snippets for this to work properly.

**Reply by vgstef** [reply]
Yes, I totally agree! That's clearly a bug. If we query custom posts, no one wants sticky posts to show up!

So, it's frustrating that Elementor closed the bug report and asked to create a feature request. It's been years that we've been waiting for a bug fix, and now they ask for enough people to vote for this feature (which is a bug) to just start being considered! How we'll people vote for this feature if it's not shown in the thread... :(   At least, this feature request should be linked in the Bug report thread!
